<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyServer</class>
 <widget class="QMainWindow" name="MyServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="historyText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>99</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ipComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
